From f5cfa6c34e8e15208ffca3a0fa39eb31f9342fc3 Mon Sep 17 00:00:00 2001
From: hmz007 <hmz007@gmail.com>
Date: Tue, 15 Dec 2020 10:33:13 +0800
Subject: [PATCH 66/73] PM / devfreq: rk3399_dmc: Fix compile error

The devfreq_event_get_edev_by_phandle() function prototype
changed (by commit 02bdbf7d09c0).

Signed-off-by: hmz007 <hmz007@gmail.com>
---
 drivers/devfreq/rk3328_dmc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/devfreq/rk3328_dmc.c b/drivers/devfreq/rk3328_dmc.c
index 9e3c87019ada..5dcea91cfae9 100644
--- a/drivers/devfreq/rk3328_dmc.c
+++ b/drivers/devfreq/rk3328_dmc.c
@@ -748,7 +748,7 @@ static int rk3328_dmcfreq_probe(struct platform_device *pdev)
 		return PTR_ERR(data->dmc_clk);
 	}
 
-	data->edev = devfreq_event_get_edev_by_phandle(dev, 0);
+	data->edev = devfreq_event_get_edev_by_phandle(dev, "devfreq-events", 0);
 	if (IS_ERR(data->edev))
 		return -EPROBE_DEFER;
 
-- 
2.31.1

